rack-test.gemspec

```diff
@@ -1,25 +1,26 @@
 # -*- encoding: utf-8 -*-
+# stub: rack-test 0.6.3 ruby lib
```

Comment:
This line makes no sense. This is autogenerated by RubyGems for .gemspec files used after installation. Moreover, it is not visible on GH, but the separator is null character, not sure what separator is used here ...